compiler/rustc_session/src/config/cfg.rs

```diff
@@ -326,6 +338,8 @@ impl CheckCfg {
 
         ins!(sym::debug_assertions, no_values);
 
+        ins!(sym::fmt_debug, empty_values).extend(FmtDebug::all());
```

Comment:
The `fmt_debug` cfg must now also be added in the [`disallow_cfgs` method](https://github.com/rust-lang/rust/blob/a971212545766fdfe0dd68e5d968133f79944a19/compiler/rustc_session/src/config/cfg.rs#L89-L148) above as to prevent user from manually setting the cfg themselves.

Don't forget to update the test: [`tests/ui/cfg/disallowed-cli-cfgs.rs`](https://github.com/rust-lang/rust/blob/a971212545766fdfe0dd68e5d968133f79944a19/tests/ui/cfg/disallowed-cli-cfgs.rs).

Reply:
I don't get how is that supposed to work. I've copied other cfgs, and got a tidy check failure:

> tidy error: unknown revision [fmt_debug_] at /checkout/tests/ui/cfg/disallowed-cli-cfgs.rs:34

Reply:
It would be nice if such lists were auto-generated from some template/script, because there's *a lot* of places where configs are mentioned. It's tedious to change them all manually, and it's impossible for me to find them all.

Reply:
We have a [documentation at the top this file](https://github.com/rust-lang/rust/blob/4d5b3b196284aded6ae99d12bcf149ffdc8ef379/compiler/rustc_session/src/config/cfg.rs#L10-L21) for when adding a new cfg.

> tidy error: unknown revision [fmt_debug_] at /checkout/tests/ui/cfg/disallowed-cli-cfgs.rs:34

You need to add `fmt_debug_` to [the list of revisions](https://github.com/rust-lang/rust/blob/a971212545766fdfe0dd68e5d968133f79944a19/tests/ui/cfg/disallowed-cli-cfgs.rs#L2-L8) in the test file.